x-pack/plugins/spaces/server/routes/api/__fixtures__/create_mock_so_repository.ts

```diff
@@ -27,13 +27,14 @@ export const createMockSavedObjectsRepository = (spaces: any[] = []) => {
       if (spaces.find((s) => s.id === id)) {
         throw SavedObjectsErrorHelpers.decorateConflictError(new Error(), 'space conflict');
       }
-      return {};
+      return { id, attributes };
     }),
     update: jest.fn((type, id) => {
-      if (!spaces.find((s) => s.id === id)) {
+      const result = spaces.find((s) => s.id === id);
+      if (!result) {
         throw SavedObjectsErrorHelpers.createGenericNotFoundError(type, id);
       }
-      return {};
+      return result[0];
```

Comment:
Returning a non-empty object now to satisfy the explicit conversion in the client.